tests/integration/ante/jws_test.go

```diff
@@ -151,26 +151,26 @@ func TestJWSExtensionOptionWithDIDBasedFeegrant(t *testing.T) {
 
 	// Create policy message with JWS extension
 	policyContent := `
-name: Bulletin Policy
 description: Base policy that defines permissions for bulletin namespaces
+name: Bulletin Policy
```

Comment:
question: is there a reason why we define description before the name in all the updated policy definitions?